docs/src/index.md

```diff
@@ -182,3 +182,58 @@ If the keyword argument `species` is not set, the resulting reaction network
 will simply name the species `S1`, `S2`,..., `SN` for a system with `N` total
 species. `params` defaults to an empty vector, so that it does not need to be
 set for systems with no parameters.
+
+## Reproducibility
+```@raw html
+<details><summary>The documentation of this SciML package was build using these direct dependencies,</summary>
```

Comment:
```suggestion
<details><summary>The documentation of this SciML package was built using these direct dependencies,</summary>
```

Reply:
Also this typo seems to be in all the PRs you are opening.

Reply:
oh no haha